apps/wallet-api-tools/package.json

```diff
@@ -23,9 +23,12 @@
     "@uiw/react-codemirror": "^4.21.22",
     "bufferutil": "^4.0.8",
     "flowbite": "^2.3.0",
+    "jotai": "^2.7.0",
+    "lucide-react": "^0.359.0",
```

Comment:
Just some informations for you nothing to change here

You can find the [lucide icons](https://icon-sets.iconify.design/lucide/) in the [iconify lib](https://icon-sets.iconify.design/) that packages almost all open source icons

And if we were to use vite, there is a [package](https://github.com/unplugin/unplugin-icons) that only loads what is necessary